### PR TITLE
#162171032 implement article readtime

### PR DESCRIPTION
**Pull request by henperi**
#### What does this PR do?
- This PR adds functionality to calculate the average length of time required to read an article.

#### Description of Task to be completed?
- When an article is created, the reading time of the article should be estimated from the content of the article.
- When users fetch an article, the reading time should also be displayed.

#### How should this be manually tested?
- Clone this Repo
- ```cd``` in the repo
- Run ```npm install``` to install all the dependencies
- Run ```npm run start:dev``` to kick-start the local server in development mode.
- Signup a new user through the signup endpoint with postman.
- Copy out the token returned.
- With the token returned, visit the article creation endpoint,and add the token to the headers on postman (access-token).
- Create the new article supplying the body, title, image, description as well as all required parameters as stated above.
- Watch and confirm that the response from the request contains the average reading time of the article.

#### What are the relevant pivotal tracker stories?
[#162171032](https://www.pivotaltracker.com/story/show/162171032)

#### Any background context you want to add?
- None

#### Screenshots

### CODE REVIEW

**Review by edogbosunny — APPROVED** [review]
Hello @henperi awesome job so far. Keep it up.

**Comment by azupatrick0** [comment]
Nice work @henperi

File: helpers/articleHelper.js [hunk 1]
```diff
@@ -57,6 +57,16 @@ const createNewTags = async (tags) => {
   return tagIds;
 };
 
+
+const calcReadingTime = (bodyText) => {
+  const matches = bodyText.match(/\S+/g);
+  const numberOfWords = matches ? matches.length : 0;
+  const averageWPM = 225;
```
Comment (thebere) [inline helpers/articleHelper.js:64]: What is `WPM`? Write it in full if it has to please.

**Review by thebere — APPROVED** [review]

File: helpers/articleHelper.js (hunk 1 above)
Comment (henperi) [reply helpers/articleHelper.js:64]: WPM means Words Per Minute. I will update it in my next PR. Thanks for the feedback.

**Pull request merged**